Code/GraphMol/RGroupDecomposition/RGroupDecomp.cpp

```diff
@@ -56,10 +56,48 @@ const std::string RLABEL = "tempRlabel";
 const std::string SIDECHAIN_RLABELS = "sideChainRlabels";
 const std::string done = "RLABEL_PROCESSED";
 
+// Various places where rgroups can be labeled
+//  the order of precedence
+const std::string RGROUP_LABELS = "RGroupLabels";
```

Comment:
Wouldn't these be better as an enum?

Reply:
I had them as strings for debugging, but an enum would work just fine.

Reply:
Worth making the change pre-merge or would you rather just leave it?